### ISSUE TITLE
Biolith causes fabric.api.biome.v1.TheEndBiomes not to generate

### ISSUE THREAD

**Issue by Trent2087** [TerraformersMC/Biolith#8]
Biolith (MC 1.20.1) causes any mod generating End Biomes with Fabric API not to generate.

As a test, I removed the "MixinTheEndBiomeSource" from the Biolith jar, and then the FAPI created End Biomes generate correctly.

**Comment by gniftygnome** [comment]
What mod are you using to test this?  I have an idea but I need a test case ouside of FAPI's own test mod.

**Comment by Trent2087** [comment]
https://www.curseforge.com/minecraft/mc-mods/eldritch-end
https://github.com/Lyof429/EndsPhantasm

**Comment by VeggeieDude** [comment]
I can attest that End's Phantasm and Biolith (bundled with traverse) crashes the game upon trying to load a world

**Comment by gniftygnome** [comment]
> I can attest that End's Phantasm and Biolith (bundled with traverse) crashes the game upon trying to load a world

This is definitely not the same issue as the one reported above.  I can't look into this without a copy of the logs.  Please submit a new issue with logs and I will take a look at it.

**Comment by gniftygnome** [comment]
I've implemented a system to scrape the Fabric biome API's End biome intake so Biolith will generate them.  However, our End biome placement system does not work in exactly the same way, so the biomes will be placed a bit differently.  If you switch back and forth between FAPI generation and Biolith generation, there may be chunk border discontinuities in the biome selection.  (Although that's not so different from how End biome placement often looks anyway...)

> https://www.curseforge.com/minecraft/mc-mods/eldritch-end
> https://github.com/Lyof429/EndsPhantasm

I've tested these mods with the updated Biolith in 1.20.1 and things look pretty similar to without Biolith to me.  I've got a few more issues to poke at and then I'll release new Biolith versions for 1.20 and up, followed by new Terraformers biome mod versions.
